web/src/OverviewSidebarOptions.tsx

```diff
@@ -42,6 +39,39 @@ const useStyles = makeStyles({
   },
 })
 
+const segmentedControlsBorderRadius = "3px"
```

Comment:
Maybe make this consistent with the updated naming? `toggleBorderRadius`? Then you can replace the one remaining `border-radius: 3px;` for `AlertsOnTopToggle`.